### ISSUE TITLE
Capture of a widget which contains an image doesn't capture the image part of the widget

### ISSUE THREAD

**Issue by gh-pap** [SachinGanesh/screenshot#114]
Expected: capture of myWidget() will include myWidget's logo
Actual: everything is captured except the logo
Additional info: see main.dart
Full disclosure: I'm not a developer, I'm just trying to prototype a business idea

Possible dup of #63 and #82 but none of those suggestions helped, so may be a new issue.

What have I tried?
I have tried using screenshotController.captureFromWidget
I have tried adjusting the delay (many values from 10ms up to 3000ms)
I have tried Image.asset('images/logo.png') [this is actually where I started but the NetworkImage has the same issue and is easier to share with you]

I have tried two ways of forcing the canvas kit to be the renderer:
first way:
> flutter run -d chrome --web-renderer canvaskit
second way (in web/index.js added just above ""):
  <script type="text/javascript">
    window.flutterWebRenderer = "canvaskit";
  </script>

but nothing I've tried works. I'm using the latest stable of both dart and flutter. Any help appreciated.
Paul
P.S. apologies, I couldn't figure out how to paste main.dart and keep the formatting, so have just attached it as a txt file

[main.dart.txt](https://github.com/SachinGanesh/screenshot/files/9306153/main.dart.txt)

**Comment by simonpham** [comment]
Running in release mode works
`--release --web-renderer canvaskit --dart-define=BROWSER_IMAGE_DECODING_ENABLED=false`

**Comment by gh-pap** [comment]
YES! It worked! You have no idea how much that helped me. I would never have figured that out on my own. Thanks so much for your help!
Cheers,
Paul

**Comment by ritheshSalyan** [comment]
For Reference [https://github.com/SachinGanesh/screenshot/issues/103](https://github.com/SachinGanesh/screenshot/issues/103)
